contracts/lifecycle/DestructibleDelayed.sol

```diff
@@ -0,0 +1,78 @@
+pragma solidity ^ 0.4.24;
+
+import "../ownership/Ownable.sol";
+
+/**
+ * @title DestructibleDelayed
+ * @dev Contract that can be selfdestructed after a delay period.
+ */
+
+
+contract DestructibleDelayed is Ownable {
+
+  // Whether a request for the self-destruction of the contract was requested.
+  bool public destructionRequested = false;
```

Comment:
As we move towards stabilizing the OpenZeppelin API for the 2.0 release, we're locking down access is some areas. This is a prime example: I'd make all of these state variables `private` (with `public` getters), to make sure derived contracts don't inadvertently change them.